wave_matching/tests/icp_tests.cpp

```diff
@@ -65,9 +51,9 @@ TEST_F(ICPTest, fullResNullMatch) {
     this->setParams(-1, perturb);
     // test and assert
     match_success = matcher->match();
-    result = matcher->getResult();
+    double diff = (matcher->getResult().matrix() - perturb.matrix()).norm();
     EXPECT_TRUE(match_success);
-    EXPECT_TRUE(isApprox(result, perturb, 0.1));
+    EXPECT_LT(diff, 0.1);
```

Comment:
In the future, once the Transformation class is done, you should be comparing two matrices using the supplied "isNear" function.  Can you make a ticket for this so we don't forget?